stock_secondary_unit/models/stock_move.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+from odoo.tools.float_utils import float_round
+
+
+class StockMove(models.Model):
+    _inherit = ["stock.move", "product.secondary.unit.mixin"]
+    _name = "stock.move"
+    _secondary_unit_fields = {
+        "qty_field": "product_uom_qty",
+        "uom_field": "product_uom",
+    }
+
+    product_uom_qty = fields.Float(
+        store=True,
+        readonly=False,
+        compute="_compute_product_uom_qty",
+        copy=True,
+        precompute=True,
+    )
+
```

Comment:
Please add this code here:

```suggestion
    secondary_uom_qty = fields.Float(
        compute="_compute_secondary_uom_qty",
        precompute=True,
        store=True,
    )

```